### ISSUE TITLE
Wave Error on Login

### ISSUE THREAD

**Issue by eStar386** [CMSgov/eAPD#2889]
When the user enters a wrong password , the wave test fails 

Steps To Reproduce 
1. Enter in a incorrect username and/or password.
2. Right click to wave this page. 

![image.png](https://images.zenhubusercontent.com/6019ab5a4e513ffb21b40c96/1660cb6c-5cfa-47c8-8b30-c85714ffa5f9)

**Comment by jeromeleecms** [comment]
This has been fixed.
